src/app/search-page/searchPage.controller.js

```diff
@@ -21,45 +21,12 @@ export default class SearchPageController {
                 this.$timeout(() => this.nearbyCities = cities);
             })
             .catch((error) => {
-                console.log(error); //TODO on next tickets
+                this.$state.go('searchError', {
```

Comment:
Since you have a routingService, why don't make use of it for this navigation? The same is true for the others redirections to 'searchError'